packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -935,8 +935,12 @@ export class EcsFargateContainerDefinition extends EcsContainerDefinitionBase im
     this.ephemeralStorageSize = props.ephemeralStorageSize;
 
     // validates ephemeralStorageSize is within limits
-    if (props.ephemeralStorageSize && (props.ephemeralStorageSize.toGibibytes() > 200 || props.ephemeralStorageSize.toGibibytes() < 21)) {
-      throw new Error('Ephemeral Storage Size must be within 21 to 200 GiB.');
+    if (props.ephemeralStorageSize) {
+      if (props.ephemeralStorageSize.toGibibytes() > 200) {
+        throw new Error(`ECS Fargate container ${id} specifies ephemeralStorageSize at ${props.ephemeralStorageSize.toGibibytes()} > 200 GB`);
```

Comment:
minor, but part of the styling:

```suggestion
        throw new Error(`ECS Fargate container '${id}' specifies 'ephemeralStorageSize' at ${props.ephemeralStorageSize.toGibibytes()} > 200 GB`);
```